settings.toml

```diff
@@ -2,6 +2,6 @@
 port = 8080
 
 [database]
-connection_url = "postgresql://localhost:5432/template_app"
+connection_url = "postgresql://localhost:5432/todo_app"
```

Comment:
@Picbridge Make sure the readme createdb command now says this and not template_app